python/speech-to-text/requirements.txt

```diff
@@ -0,0 +1,5 @@
+aiofile==3.8.1
+amazon-transcribe==0.6.1
+asyncio==3.4.3
+azure-cognitiveservices-speech==1.24.0
```

Comment:
I tried to build this, but got this error:

```shell
$ docker run --rm --interactive --tty --volume $PWD:/usr/code openruntimes/python:v2-3.8 sh /usr/local/src/build.sh
Unable to find image 'openruntimes/python:v2-3.8' locally
v2-3.8: Pulling from openruntimes/python
9621f1afde84: Already exists 
abed0206f391: Already exists 
03a85617f558: Pull complete 
45a080554e99: Pull complete 
e760f925725f: Pull complete 
855d2a3c5cb4: Pull complete 
521a0205f4d8: Pull complete 
4aafa920114f: Pull complete 
c02e9359f9f8: Pull complete 
4f4fb700ef54: Pull complete 
7574f33118ad: Pull complete 
Digest: sha256:fb647181b5e556c8ed1ee1b5158d014b5136137284f982fe4c17fff390c2edc0
Status: Downloaded newer image for openruntimes/python:v2-3.8
Collecting aiofile==3.8.1
  Downloading aiofile-3.8.1.tar.gz (18 kB)
  Preparing metadata (setup.py) ... done
Collecting amazon-transcribe==0.6.1
  Downloading amazon_transcribe-0.6.1-py3-none-any.whl (38 kB)
Collecting asyncio==3.4.3
  Downloading asyncio-3.4.3-py3-none-any.whl (101 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 101.8/101.8 KB 24.7 MB/s eta 0:00:00
ERROR: Could not find a version that satisfies the requirement azure-cognitiveservices-speech==1.24.0 (from versions: none)
ERROR: No matching distribution found for azure-cognitiveservices-speech==1.24.0
WARNING: You are using pip version 22.0.4; however, version 22.3.1 is available.
You should consider upgrading via the '/usr/builds/runtime-env/bin/python3 -m pip install --upgrade pip' command.
```

I tried with python 3.8, 3.9, and 3.10. How did you build this?

Reply:
I did this with python 3.9

Reply:
Did you run using open runtimes? What was your exact command to build?

Reply:
> Did you run using open runtimes? What was your exact command to build?

No, can you please share the docs/steps for that?

Also, I was checking contributing guide but it is showing an [error](https://github.com/open-runtimes/examples/blob/main/CONTRIBUTING.md)